src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1757,6 +1757,10 @@ format layout_optimizer::get_preferred_format(program_node& node) {
         // Set default format for issue 92967/98750
         // TODO: will remove when arg_max_min_ref supports blocked format
         expected = format::get_default_format(node.get_input_layouts()[0].get_rank(), false, false);
+    } else if (node.is_type<fully_connected>()) {
+        if (use_onednn_impls) {
+            expected = format::bfyx;
+        }
```

Comment:
What about querying onednn layout for fc?

Reply:
Agree, I'll follow up.